website/content/en/pre-docs/deploy-guide/_index.md

```diff
@@ -0,0 +1,165 @@
+
+### Minimum Kubernetes Version
+
+Karpenter supports the [same set of Kubernetes versions as EKS supports](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html) for new clusters. Karpenter strongly encourages a posture of promptly adopting new versions of Kubernetes.
+
+### Service Account
+    - `Karpenter` Service Account
+
+The `Karpenter` service account needs sufficient cluster permissions to, for example, add new nodes and bind pods to nodes. Review the ["karpenter-controller" `role` and `ClusterRole`](rbac.yaml) from the default helm chart to understand what permissions the Karpenter service account needs. The Karpenter helm chart includes other roles (`karpenter-webhook`), but the "karpenter-controller" resources require uniquely broad access.
+
+In part:
+
+```
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: karpenter-controller
+rules:
+[...]
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - create
+[...]
+```
+
+
+### IAM Roles
+- `Karpenter` Role
+- `KarpenterNode` Role
+
+Generally, Karpenter needs to authenticate with the AWS EC2 API to create new instances. This is best done with IRSA, IAM Roles for Service Accounts. IRSA naturally extends the `Karpenter` service account to handle interacting with AWS APIs. The getting started guide describes creating a cluster service account mapped to an IAM role with sufficient permissions. 
+
+Karpenter may alternatively be configured to authenticate with AWS using the `kube2iam` package, or with locally stored Access Keys. Both practices are discouraged. 
+
+The provisioned instances must comparable permissions to a standard EKS self-managed node. For example, permission to configure instance networking interfaces. The getting started guide again provides a useful starting point in [the cloudformation template](../getting-started/cloudformation.yaml).
+
+### Tagged Subnets
+
+Karpenter needs to discover VPC (virtual private cloud) subnets on AWS, such that requests for new instances are places on the proper subnet, enabling communication with the control plane. Karpenter discovers subnets using resource tags. 
+
+The tags must have this form:
+- key: `kubernetes.io/cluster/${CLUSTER_NAME}`
+- value: "" (empty string)
+
+In some situations, EKS may automatically apply this tag to subnets. Do not rely on EKS to do this. 
+
+### `eksctl`
+
+The getting started guide uses `eksctl`. This tool is officially supported and sponsored by AWS. However, equivalent EKS clusters may be created using CloudFormation, or even the web console. 

```

Comment:
```suggestion
The getting started guide uses `eksctl`. This tool is officially supported and sponsored by AWS. However, equivalent EKS clusters may be created using CloudFormation, Terraform, or even the web console. 

```
A majority of our users use terraform for cluster creation so we should call it out